src/main/323C.c

```diff
@@ -471,7 +471,65 @@ void load_vram_rect_ptrs(void)
     vram_rect_ptr = &vram_rect_ptrs[0];
 }
 
-INCLUDE_ASM("asm/us/main/nonmatchings/323C", func_80015ECC);
+extern s32 D_8016DEA8;
+extern s32 D_8016EEA8;
```

Comment:
Worth pulling these out as named symbols? Or still figuring out exactly what they do (seems like this area holds graphics data)

Reply:
I named them